sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
actually, I take that back - I was doing it wrong.
You can move sc-cli.go int "cmd" dir and it works just fine.
Just one code change to get it to compile.